contributors.yaml

```
@@ -0,0 +1,276 @@
+- given-names: Timothy
+  family-names: Hallett
+  orcid: "https://orcid.org/0000-0002-9681-0424"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/timothy.hallett"
+  github-username: tbhallett
+  role: Joint lead epidemiology
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Andrew
+  family-names: Phillips
+  orcid: "https://orcid.org/0000-0003-2384-4807"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/5430"
+  github-username: andrew-phillips-1
+  role: Joint lead epidemiology
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Asif
+  family-names: Tamuri
+  orcid: "https://orcid.org/0000-0001-6633-3789"
+  affiliation: "University College London"
+  website: "https://www.homepages.ucl.ac.uk/~ucbtaut/"
+  github-username: tamuri
+  role: "Lead developer"
+  contributions:
+    - Software development
+- given-names: Joseph
+  family-names: Mfutso-Bengo
+  orcid: "https://orcid.org/0000-0001-9396-3700"
+  affiliation: "Kamuzu University of Health Sciences"
+  website: "https://www.researchgate.net/profile/Joseph-Mfutso-Bengo"
+  role: "Lead policy translation"
+  contributions:
+    - Policy translation
+- given-names: Tim
+  family-names: Colbourn
+  orcid: "https://orcid.org/0000-0002-6917-6552"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/3084"
+  github-username: timcolbourn
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Joseph
+  family-names: Collins
+  orcid: "https://orcid.org/0000-0003-4801-7218"
+  affiliation: "University College London"
+  website: "https://profiles.ucl.ac.uk/70734"
+  github-username: joehcollins
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Iwona
+  family-names: Hawryluk
+  orcid: "https://orcid.org/0000-0001-6919-4366"
+  affiliation: "Imperial College London"
+  website: "https://www.imperial.ac.uk/people/i.hawryluk19"
+  github-username: ihwaryluk
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Britta
+  family-names: Jewell
+  orcid: "https://orcid.org/0000-0003-4275-8509"  # to check
+  affiliation: "Imperial College London" # to check
+  website: https://www.imperial.ac.uk/people/b.jewell
+  github-username: brittajewell
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Ines
+  family-names: Li Lin
+  orcid: "https://orcid.org/0000-0002-1945-3208"
+  affiliation: "University College London"
+  github-username: inesll
+  contributions:
+    - Epidemiology and modelling
+    - Software development
+- given-names: Tara
+  family-names: Mangal
+  orcid: "https://orcid.org/0000-0001-9222-8632"  # to check
+  affiliation: "Imperial College London / University of York"  # to check
```

```suggestion
  orcid: "https://orcid.org/0000-0001-9222-8632"
  affiliation: "Imperial College London"
```
Confirmed above ORCID and affliation with Tara